tools/SetupFlow/DevHome.SetupFlow/Services/SetupFlowOrchestrator.cs

```diff
@@ -171,12 +170,53 @@ private bool CanGoToPreviousPage()
         return HasPreviousPage && CurrentPageViewModel.CanGoToPreviousPage;
     }
 
+    // Teaching tip needs to be open and visible to show up.
+    [ObservableProperty]
+    private bool _isTeachingTipOpen;
+
+    [ObservableProperty]
+    private bool _isTeachingTipVisible;
+
     [RelayCommand(CanExecute = nameof(CanGoToNextPage))]
     public async Task GoToNextPage()
     {
+        IsTeachingTipVisible = false;
+        IsTeachingTipOpen = false;

```

Comment:
We could probably make either another property or a method that sets both of these to the same value. Since it looks like we need to keep them in sync, this way we don't accidentally change them in the future. Your reply to Kristen could probably be a comment too stating that it's a win ui limitation.